src/components/typography/index.ts

```diff
@@ -0,0 +1,20 @@
+// @ts-nocheck - bugged with astro
```

Comment:
no clue what this is about, what you can just use an astro file for this and put this code in the frontmatter